bot/resources/tags/comparison.md

```diff
@@ -0,0 +1,11 @@
+#### Assignment vs. Comparison
```

Comment:
```suggestion
**Assignment vs. Comparison**
```
"###" doesn't work for formatting on Discord. This is how you make it bold (we sadly can't use a native embed title, due to the way that tags are made).